shared/Microsoft.Extensions.ActivatorUtilities.Sources/sharedsources.props

```diff
@@ -0,0 +1,8 @@
+<Project>
+  <ItemGroup>
+    <Compile Include="$(NuspecBasePath)\..\Microsoft.Extensions.ParameterDefaultValue.Sources\ParameterDefaultValue.cs">
```

Comment:
I tried hard but wasn't able to make it work. My other though was just to leave them slip and reference both where we reference ActivatorUtils